CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ All notable changes to this project will be documented in this file.
 ## [Unreleased]
 
 ### Added
+- Support for reading MWAX/birli fits files
```

Comment:
This is uvfits not just fits right? Pyxie's reader is for the raw fits files.

Reply:
it is. I've updated this in https://github.com/RadioAstronomySoftwareGroup/pyuvdata/pull/1086/commits/2905273e3c724fdb40789345bb91c537dd78b496 and added a `changed` entry to reflect what uvfits reader does now.